package.js

```diff
@@ -4,7 +4,7 @@ Package.describe({
   // Brief, one-line summary of the package.
   summary: 'ElasticSearch utility wrapper for mongoDB integration and sync',
   // URL to the Git repository containing the source code for this package.
-  git: 'https://github.com/toystars/elasticsearch-sync',
+  git: 'https://github.com/Rvach/elasticsearch-sync',
```

Comment:
Sorry, it is for internal usage only. I'll change it back as soon as your repo is fixed.